build/lib/extensions.ts

```diff
@@ -343,6 +355,7 @@ export function packageLocalExtensionsStream(forWeb: boolean, disableMangle: boo
 				const extensionName = path.basename(extensionPath);
 				return { name: extensionName, path: extensionPath, manifestPath: absoluteManifestPath };
 			})
+			.filter(({ name }) => extensionFilter ? extensionFilter.indexOf(name) >= 0 : true)
```

Comment:
Can `glob.sync` operate on `extensionFilter` so that double packaging is avoided. It also aligns with the fact we know the list of extensions that should act as source in the `packageLocalNativeExtensionsStream` case.

Reply:
I separated things out so now there are mutually exclusive tasks and also a big task that does it all (for non-ci tasks like localization).

Let me know what you think!